modules/apps/digital-signature/digital-signature-lang/src/main/resources/content/Language.properties

```diff
@@ -21,6 +24,7 @@ recipient-full-name=Recipient Full Name
 rsa-private-key=RSA Private Key
 the-document-does-not-have-a-preview=The document does not have a preview.
 this-field-exceeded-x-characters=This field exceeded {0} characters.
+this-file-extension-is-not-supported-by-docusign=This file extension is not supported by DocuSign
```

Comment:
This is a sentence, insert a period at the end.
**This file extension is not supported by DocuSign.**